content/installation/installing_manager.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: bt_wiki
+title: Installing Cloudify Manager
+category: Installation
+draft: false
+weight: 100
+---
+A Cloudify Manager is a compute host that runs the Cloudify Management service runs. For version 4.3 and higher, you can install a single rpm file that installs Cloudify Manager with all of its dependencies. This installation also includes plugins and blueprints so you can run a demonstration of the Cloudify features. You can then install the Cloudify CLI to manage your Cloudify Manager remotely.
```

Comment:
double "runs" in the first sentence

Reply:
`You can then install the Cloudify CLI` - previous sentence talks about installing the manager, so maybe emphasize that you install the CLI somewhere else?

Reply:
btw, is the part about plugins and blueprints actually true?

Reply:
@tehasdf Not yet, but should be for 4.3 (possibly).
@bmansheim I'd wait before actually adding it to the documentation, as it might turn out false.